src/main/java/com/team1816/lib/subsystems/SwerveModule.java

```diff
@@ -109,61 +111,57 @@ public void setDesiredState(SwerveModuleState desiredState, boolean isOpenLoop)
         azimuthMotor.set(ControlMode.Position, azimuthDemand);
     }
 
-    public SwerveModuleState getState() {
-        double velocity =
-            DriveConversions.convertTicksToMeters(
-                driveMotor.getSelectedSensorVelocity(0)
-            ) *
-            10;
-        Rotation2d angle = Rotation2d.fromDegrees(
+    public SwerveModuleState getActualState() {
+        driveActual =
+            DriveConversions.ticksToMeters(driveMotor.getSelectedSensorVelocity(0)) * 10;
+        azimuthActual =
             DriveConversions.convertTicksToDegrees(
                 azimuthMotor.getSelectedSensorPosition(0) -
                 mConstants.kAzimuthEncoderHomeOffset
-            )
-        );
-        return new SwerveModuleState(velocity, angle);
+            );
+        Rotation2d angleActual = Rotation2d.fromDegrees(azimuthActual);
+        motorTemp = driveMotor.getTemperature(); // Celsius
+        return new SwerveModuleState(driveActual, angleActual);
     }
 
     @Override
-    public String getSubsystemName() {
-        return mConstants.kName;
+    public double getMotorTemp() {
+        return motorTemp;
     }
 
     @Override
-    public double getAzimuthActual() {
-        return azimuthMotor.getSelectedSensorPosition(0);
+    public String getName() {
```

Comment:
I think the reason we do getSubsystemName is because getName is a deprecated method in the `Sendable` class.

Reply:
yes correct